feed.php

```diff
@@ -1,113 +1,191 @@
+    }
+    return $images;
+    //return ['hola', 'images'];
+}
+
+function getProducts() {
+    global $storeID, $currency, $baseCurrencyCode, $sizeLimit, $lastProductId;
+
+    $products = Mage::getResourceModel('catalog/product_collection')->setStore($storeID);
+    $products->addAttributeToFilter('status', 1);// get enabled prod
+    $products->addFieldToFilter('entity_id', array('gt' => $lastProductId));
+    $products->setOrder('entity_id', Varien_Data_Collection::SORT_ORDER_ASC);
+    $products->setPageSize($sizeLimit)->load();
 
-      //$products = Mage::getModel('catalog/product')->getCollection();
-      $products = Mage::getResourceModel('catalog/product_collection')->setStore($storeID);
-      $products->addAttributeToFilter('status', 1);// get enabled prod
-      $prodIds = $products->getAllIds();
+    $prodIds = $products->getLoadedIds();
 
-      $prods = array();
+    $prods = array();
 
-      foreach ($prodIds as $productId) {
-          $product = Mage::getModel('catalog/product');
-          $product->load($productId);
+    foreach ($prodIds as $productId) {
+        $product = Mage::getModel('catalog/product');
+        $product->load($productId);
 
-          $prodData = array();
+        $lastProductId = $productId;
 
-          $prodData['storeId'] = $storeID;
-          $prodData['title'] = html_entity_decode(strip_tags($product->getName()));
-          $prodData['description'] = html_entity_decode(strip_tags($product->getDescription()));
+        $prodData = array();
 
-          // get parent product ID if exist and get url of parent
-          $parentIds = Mage::getResourceSingleton('catalog/product_type_configurable')->getParentIdsByChild($productId);
-          $groupedParentsIds = Mage::getResourceSingleton('catalog/product_link')
-                     ->getParentIdsByChild($productId, Mage_Catalog_Model_Product_Link::LINK_TYPE_GROUPED);
-          if($parentIds){
+        $prodData['storeId'] = $storeID;
+        $prodData['title'] = html_entity_decode(strip_tags($product->getName()));
+        $prodData['description'] = html_entity_decode(strip_tags($product->getDescription()));
+
+        // get parent product ID if exist and get url of parent
+        $parentIds = Mage::getResourceSingleton('catalog/product_type_configurable')->getParentIdsByChild($productId);
+        $groupedParentsIds = Mage::getResourceSingleton('catalog/product_link')
+            ->getParentIdsByChild($productId, Mage_Catalog_Model_Product_Link::LINK_TYPE_GROUPED);
+        if($parentIds){
             $artno = intval($parentIds[0]);
             $url =  Mage::getModel('catalog/product')->load($artno)->getUrlInStore();
-          }else if($groupedParentsIds){
+        }else if($groupedParentsIds){
             $artno = intval($groupedParentsIds[0]);
             $_p = Mage::getModel('catalog/product')->load($artno);
             $url =  $_p->getUrlInStore();
             $prodData['title'] = $_p->getName();
```

Comment:
should be (also override description)

    $prodData['title'] = html_entity_decode(strip_tags($_p->getName());
    $prodData['description'] = html_entity_decode(strip_tags($_p->getDescription()));